convert2rhel/subscription.py

```diff
@@ -973,8 +973,10 @@ def check_needed_repos_availability(repo_ids_needed):
         if repo_id not in avail_repos:
             # TODO: List the packages that would be left untouched
             loggerinst.warning(
-                "%s repository is not available - some packages"
-                " may not be replaced and thus not supported." % repo_id
+                "Some repositories are not available: %s. Some packages may not be replaced with"
+                "their corresponding RHEL packages when converting. The converted system will be"
+                "running a mixture of packages from RHEL and from your current distribution."
+                "Try enabling the repositories and running convert2rhel again." % repo_id
```

Comment:
_"Try enabling the repositories and running convert2rhel again."_
I see no way for the user to enable the repositories, making this suggestion confusing.